docs/platform-integration/essentials-index.md

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
I'd delete this for now. I can maybe see the value in there being a good overview page at some point, but as you've said, it's just a link farm right now.